src/services/api/api.contoller.test.ts

```diff
@@ -140,29 +136,31 @@ describe('ApiController', () => {
     it('should link a Hive account', async () => {
       const jwtToken = 'test_jwt_token';
 
+      const user_id = 'test_user_id'
+      const user = await authService.createDidUser('bob', user_id)
```

Comment:
TODO validation `'bob'` is a valid DID in auth service

Reply:
done